go/api/config.go

```diff
@@ -0,0 +1,309 @@
+// Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+import "github.com/aws/glide-for-redis/go/glide/protobuf"
+
+const (
+	defaultHost = "localhost"
+	defaultPort = 6379
+)
+
+// NodeAddress represents the host address and port of a node in the cluster.
```

Comment:
[Go doc comments](https://tip.golang.org/doc/comment) are supposed to include the name of the type or function they are documenting in the first sentence, so the docs are slightly different than the other clients to accommodate that. I've tried to keep them as close possible to the other clients still.